ts/data/case.ts

```diff
@@ -184,6 +189,10 @@ export const caseDefinition: TableDefinition<Case, Test> = {
               (kase.timepoint ? " " + kase.timepoint : "")
           )
         );
+        tooltipInstance.addTarget(
+          tumourDetailDiv,
+          document.createTextNode("Timepoint")
```

Comment:
Hmm... This line shows 2-3 different attributes, and timepoint isn't always included. Can you change this tooltip to look like below?

> Tissue Origin: XY
> Tissue Type: Z
> Timepoint: asdf

and omit the timepoint line if the sample has no timepoint